src/mount/mount.ceph.c

```diff
@@ -425,24 +425,41 @@ static void ceph_mount_info_free(struct ceph_mount_info *cmi)
 	free(cmi->cmi_conf);
 }
 
-static int finalize_options(struct ceph_mount_info *cmi)
+static int append_secret_option(struct ceph_mount_info *cmi)
 {
-	int pos;
+	int pos = strlen(cmi->cmi_opts);
```

Comment:
Can cmi->cmi_opts be NULL here?  Previously strlen() call was conditioned on `cmi->cmi_secret[0] || is_kernel_secret(cmi->cmi_name)`.

Reply:
No, it can't be NULL at this point. The last thing that parse_options does is ensure that.